app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,83 @@
+# :reek:InstanceVariableAssumption
+class TeamsController < AuthenticatedController
```

Comment:
This is actually the `GroupsController` with a rename....but I guess the changes were big enough that git/Github thinks they are different files